### ISSUE TITLE
Project Limit Request: Libdeeplake - 100 GB (bumped to 20 GB)

### ISSUE THREAD

**Issue by davidbuniat** [pypi/support#3238]
### Project URL

https://pypi.org/project/libdeeplake/

### Does this project already exist?

- [X] Yes

### New limit

100

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI

### About the project

We are shipping deeplake (https://github.com/activeloopai/deeplake) backend binary named as libdeeplake

So far we expect 100GB should be totally fine.  

Thanks in advance for the awesome work!

### How large is each release?

Total size ~200MB (single wheel size 10-20MB) 

### How frequently do you make a release?

Every week

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by davidbuniat** [comment]
Hi @cmaureir, would really appreciate if you could review the proposal. Please let us know if there is anything we should do on our side!

**Comment by di** [comment]
Can you add more detail about why the binary is so large, and why you need the weekly release frequency? Thanks!

**Comment by levongh** [comment]
Hi @di the binaries are as big because of the fact that we are statically inking a couple of could provider SDKs (e.g [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp, [Azure-SDK]https://github.com/Azure/azure-sdk)

The reason for having frequent releases is some user-critical bug fixes. But we are going to decrease the end binary size and have less frequent releases

**Comment by levongh** [comment]
hey @di, Is there any action item from our side that is missing? The storage limit issue is blocking our releases. Thanks!

**Comment by levongh** [comment]
Hi @cmaureir @di, We would really love to hear back any answer, so there will be clarity in our release planning and moving forward strategies.

Is there any information or action items from our side?
I really appreciate any help you can provide. Thanks!

**Comment by cmaureir** [comment]
Hey @levongh :wave:
I personally don't feel like a 10x increase on the project size is something common. Doubling or tripling the size of a project is something I have personally done in the past.
Additionally, if your binaries are so large, I'd say something needs to be rethink regarding the release process. A so-frequent release cycle, maybe works well with removing old versions if it's too critical for people to get the last releases.
Statically linking such large SDKs does sounds like a problematic detail for shipping the module on PyPI, have you consider dynamically linking the module? maybe the dependency (`aws-sdk-cpp`) can be placed on PyPI (if it's not there) and you can use that dependency, which I'm certain doesn't have a weekly release.

For unlocking your release process, I have increase the project size to 20G, but I'd encourage you can discuss the previous comments with the team, to look for alternatives. Only a few projects on PyPI have a space larger 100G, that's why I'm so cautious.

**Comment by levongh** [comment]
hey @cmaureir,

Thanks for the support, we will revisit the binary construction process!

**Comment by pradyunsg** [comment]
(added context that the limit was bumped to 20 GB already, as part of this issue's discussion; for posterity)

**Comment by pradyunsg** [comment]
> we will revisit the binary construction process!

That's great to hear! Let us know if revisiting this binary construction process works out successfully for y'all. :)

I'll close this out for now, since we've bumped the limit here and unblocked you for now. If there's a need for a future limit increase, please file a new issue and add relevant context there.

**Comment by davidbuniat** [comment]
thanks a lot @pradyunsg @cmaureir
